web/.nginx/routes.js

```diff
@@ -0,0 +1,70 @@
+export default [
```

Comment:
Can you delete this file?

It was in `gitignore` but we now save it in another place.